Content/Projectiles/Utility/ExitPortal.cs

```diff
@@ -49,6 +54,73 @@ public override void AI()
 		}
 
 		Lighting.AddLight(Projectile.Center, TorchID.Red);
+
+		if (ItemMagnetTimer++ == 60)
+		{
+			for (int i = 0; i < Main.maxItems; ++i)
+			{
+				Item item = Main.item[i];
+
+				// Invalid items to teleport (inactive or takes up a lot of space, might push items far away)
+				if (!item.active || item.IsACoin || item.type == ModContent.ItemType<HealingPotionPickup>() || item.type == ModContent.ItemType<ManaPotionPickup>())
```

Comment:
Best also off-lift into an `MagnetItems` method.
```suggestion
		if (ItemMagnetTimer++ == 60)
		{
			foreach (Item item in Main.ActiveItems)
			{
				// Invalid items to teleport (if takes up a lot of space, might push items far away, etc.)
				if (item.IsACoin || item.type == ModContent.ItemType<HealingPotionPickup>() || item.type == ModContent.ItemType<ManaPotionPickup>())
```

Reply:
I used a for loop instead of a foreach because the doc for `item.whoAmI` said "unused". Unsure what "unused" means, so I relied on `i` instead of `item.whoAmI`. Can reorganize next week.